### PR TITLE
Repository: Chores.

### PR DESCRIPTION
**Pull request by Gacko**
### What does this PR do?

(Please set a descriptive PR title. Use this space for additional explanations.)

### How does it look like?

(Please add anything that represents the change visually. Screenshots, output, logs, ...)

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Should this change be mentioned in the release notes?

- [x] CHANGELOG.md has been updated (if it exists)

### CODE REVIEW

**Review by njuettner — APPROVED** [review]
ok to me cc @nprokopic

File: Makefile.development.mk [hunk 1]
```diff
@@ -18,8 +13,7 @@ endif
 template: ## Output the rendered Helm template
 	$(eval CHART_DIR := "helm/cluster")
 	$(eval HELM_RELEASE_NAME := $(shell yq .global.metadata.name ${CHART_DIR}/${CI_FILE}))
-	@cd ${CHART_DIR} && \
-		helm template -f ${CI_FILE} --debug ${HELM_RELEASE_NAME} .
+	@helm template ${HELM_RELEASE_NAME} ${CHART_DIR} --values ${CI_FILE} --debug
```
Comment (nprokopic) [inline Makefile.development.mk:16]: This breaks `make template`

```
make template
install.go:214: [debug] Original chart version: ""
install.go:231: [debug] CHART PATH: /Users/nikola/src/giantswarm/cluster/helm/cluster

Error: open ci/ci-values.yaml: no such file or directory
helm.go:84: [debug] open ci/ci-values.yaml: no such file or directory
make: *** [template] Error 1
```

**Review by nprokopic — CHANGES_REQUESTED** [review]

File: Makefile.development.mk (hunk 1 above)
Comment (nprokopic) [inline Makefile.development.mk:16]:
```suggestion
	@helm template ${HELM_RELEASE_NAME} ${CHART_DIR} --values ${CHART_DIR}/${CI_FILE} --debug
```

Comment (Gacko) [reply Makefile.development.mk:16]: Ah, yeah, my mistake! Need to prepend the "${CHART_DIR}"

Comment (Gacko) [reply Makefile.development.mk:16]: Or you do it 😬

**Review by nprokopic — APPROVED** [review]

**Pull request merged**